client/src/app/pages/dashboard/maps/upload-form/map-upload-form.component.ts

```diff
@@ -25,6 +25,8 @@ export interface ImageFilePreview {
   file: File;
 }
 
+var youtubeRegex = /[a-zA-Z0-9_-]{11}/;
```

Comment:
```suggestion
const youtubeRegex = /[a-zA-Z0-9_-]{11}/;
```

`var` should never be used.